src/Compilation/Config.zig

```diff
@@ -433,6 +433,9 @@ pub fn resolve(options: Options) ResolveError!Config {
 
     const debug_format: DebugFormat = b: {
         if (root_strip and !options.any_non_stripped) break :b .strip;
+        if (options.debug_format) |options_format| {
+            break :b options_format;
+        }
```

Comment:
```suggestion
        if (options.debug_format) |x| break :b x;
```

Minor style nitpick to be consistent with the surrounding code.